src/Pages/ContentListTable/components/StatusIcon.tsx

```diff
@@ -43,31 +71,50 @@ const StatusIcon = ({ status, error }: Props) => {
       );
     case 'Invalid':
       return (
-        <Tooltip position='top-start' content={error}>
           <Flex alignContent={{ default: 'alignContentCenter' }} direction={{ default: 'row' }}>
             <FlexItem spacer={{ default: 'spacerSm' }}>
               <ExclamationCircleIcon color={red} />
             </FlexItem>
             <FlexItem>
-              <StatusText color='red'>Invalid</StatusText>
+              <Popover
+                aria-label="invalid popover"
+                alertSeverityVariant='danger'
+                headerContent="Invalid"
+                headerIcon={<ExclamationCircleIcon/>}
+                bodyContent={popoverDescription(error, failedIntrospectionsCount, lastIntrospectionTime)}
+                position='left'
+                footerContent={popoverFooter(retryHandler, uuid)}
+              >
+                <Button variant="link" isInline>
+                    <StatusText color='red' isLink>Invalid</StatusText>
+                  </Button>
+             </Popover>
             </FlexItem>
           </Flex>
-        </Tooltip>
       );
     case 'Unavailable':
       return (
-        <Tooltip position='top-start' content={error}>
           <Flex alignContent={{ default: 'alignContentCenter' }} direction={{ default: 'row' }}>
             <FlexItem spacer={{ default: 'spacerSm' }}>
               <ExclamationTriangleIcon color={gold} />
             </FlexItem>
             <FlexItem>
-              <StatusText color='gold'>Unavailable</StatusText>
+              <Popover
+                aria-label="unavailable popover"
+                alertSeverityVariant='warning'
+                headerContent="Unavailable"
+                headerIcon={<ExclamationTriangleIcon/>}
+                bodyContent={popoverDescription(error, failedIntrospectionsCount, lastIntrospectionTime)}
+                position='left'
+                footerContent={popoverFooter(retryHandler, uuid)}
+              >
+                <StatusText color='gold' isLink>Unavailable</StatusText>
+              </Popover>
             </FlexItem>
           </Flex>
-        </Tooltip>
       );
     case 'Pending':
```

Comment:
case '':

Reply:
This shouldn't be needed if we fix [this](https://issues.redhat.com/browse/HMS-1564)